app/assets/scripts/utils/field-report-constants.js

```diff
@@ -0,0 +1,150 @@
+'use strict';
+
+export const status = [
+  {
+    label: 'Early Warning',
+    value: 'early-warning',
+    description: 'Early warning message for an upcoming disaster.'
+  },
+  {
+    label: 'Event',
+    value: 'event',
+    description: 'First report for this disaster.'
+  }
+];
+
+export const countries = [
```

Comment:
I know these are just here for the sake of development, but would it be helpful to have endpoints for all of these (ie countries, disaster types, etc)? Some of them, like `Events`, will need API endpoints of course.

Reply:
Ideally that would be good yes, but given the development effort I think that implementation will depend on how likely is this to change. Countries for example, shouldn't change, so it might be ok to keep as a constant.